common/Services/NotificationService.cs

```diff
@@ -0,0 +1,240 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using System.Threading.Tasks;
+using CommunityToolkit.Mvvm.Input;
+using CommunityToolkit.WinUI;
+using CommunityToolkit.WinUI.Behaviors;
+using DevHome.Common.Contracts;
+using DevHome.Common.Environments.Helpers;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.Windows.AppLifecycle;
+using Microsoft.Windows.AppNotifications;
+using Microsoft.Windows.AppNotifications.Builder;
+using Serilog;
+using Windows.ApplicationModel.Activation;
+
+namespace DevHome.Common.Services;
+
+public class NotificationService
+{
+    private readonly ILogger _log = Log.ForContext("SourceContext", nameof(NotificationService));
+
+    private readonly IWindowsIdentityService _windowsIdentityService;
+
+    private readonly string _componentName = "NotificationService";
+
+    private readonly string _hyperVText = "Hyper-V";
+
+    private readonly string _microsoftText = "Microsoft";
+
+    private readonly StringResource _stringResource;
+
+    private readonly Microsoft.UI.Dispatching.DispatcherQueue _dispatcher;
+
+    private StackedNotificationsBehavior? _notificationQueue;
+
+    public NotificationService(IWindowsIdentityService windowsIdentityService)
+    {
+        _windowsIdentityService = windowsIdentityService;
+        _dispatcher = Microsoft.UI.Dispatching.DispatcherQueue.GetForCurrentThread();
+        _stringResource = new StringResource("DevHome.Common.pri", "DevHome.Common/Resources");
+    }
+
+    public bool ShowHyperVAdminWarningToast()
+    {
+        // Temporary toast notification to inform the user that they are not in the Hyper-V admin group.
+        // In the future we'll use an admin process from Dev Home to add the user to the group.
+        var toast = new AppNotificationBuilder()
+           .AddText("Warning")
```

Comment:
Should be localized.

Reply:
From the previous file; due to be completely removed in a later PR.